src/core/core/model/osint_source.py

```
@@ -946,6 +948,96 @@ def import_osint_sources_from_json(cls, json_data) -> list[str]:
+                        group.osint_sources.append(source)
+
+            db.session.commit()
+        except Exception:
+            db.session.rollback()
+            raise
+
+        for source in created_sources:
+            try:
+                if source.schedule_osint_source() is not True:
+                    logger.warning("Curated OSINT source %s will be scheduled by queue reconciliation", source.id)
+            except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)